docs/reference/field-types/url.md

```diff
@@ -0,0 +1,43 @@
+# `url`
+
+`url` adds an editable URL field to the schema.
+
+Apostrophe will detect common mistakes, including leaving off `https://`. Common XSS attack vectors are laundered and discarded. Only "safe" URL schemes, e.g., `http`, `https`, `ftp` and `mailto`, are permitted.
```

Comment:
"Common XSS attack vectors are laundered and discarded. Only "safe" URL schemes, e.g., `http`, `https`, `ftp` and `mailto`, are permitted."